src/overlays/actors/ovl_Dm_Opstage/z_dm_opstage.h

```diff
@@ -4,23 +4,24 @@
 #include "global.h"
 
 #define DMOPSTAGE_GET_TYPE(thisx) (((thisx)->params & 0xFF))
-#define DMOPSTAGE_GET_08(thisx) (((thisx)->params >> 8) & 0xFF)
+#define DMOPSTAGE_GET_ACTORACTION(thisx) (((thisx)->params >> 8) & 0xFF)
 
 struct DmOpstage;
 
 typedef void (*DmOpstageActionFunc)(struct DmOpstage*, GlobalContext*);
+
 typedef struct DmOpstage {
     /* 0x000 */ DynaPolyActor dyna;
-    /* 0x15C */ UNK_TYPE1 pad_15C[0x44];
+    /* 0x15C */ UNK_TYPE1 pad15C[0x44];
```

Comment:
Hmm, one thing that is missing from the style guide is how to format these. I note that there are over 20k unks and < 3k unk_s.

Reply:
We don't really use snake case for any other variable unless its screaming snake. I thought the only reason people left the underscore was because that was what m2c gave us and the left it, which I was okay with until documentation since they would often be given a completely different name at that point anyway.

Reply:
OoT consistently uses underscores for unks.